test/unit/nxos/mock_data/show_bgp_sessions_vrf_all.json

```diff
@@ -0,0 +1,84 @@
+{
+  "ins_api": {
+    "type": "cli_show",
+    "version": "1.0",
+    "sid": "eoc",
+    "outputs": {
+      "output": {
+        "input": "show bgp sessions vrf all",
+        "msg": "Success",
+        "code": "200",
+        "body": {
+          "totalpeers": "3",
+          "totalestablishedpeers": "3",
+          "localas": "11111",
+          "TABLE_vrf": {
+            "ROW_vrf": [
```

Comment:
I see this on a Nexus having just the default routing table:

```json
{
  "totalpeers": 4,
  "totalestablishedpeers": 4,
  "localas": 65005,
  "TABLE_vrf": {
    "ROW_vrf": {
      "vrf-name-out": "default",
      "local-as": 65005,
      "vrfpeers": 4,
      "vrfestablishedpeers": 4,
```

which means ```dict``` instead of ```list```.